gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/DatastoreOptionsTest.java

```diff
@@ -34,6 +34,7 @@
 public class DatastoreOptionsTest {
 
   private static final String PROJECT_ID = "project_id";
+  private static final int PORT = LocalGcdHelper.findOpenPort();
```

Comment:
Also, because DEFULT might not be available we can change the method to accept a default value that would
be used when we could not find an available port (instead of throwing an exception).